rubocop.yml

```diff
@@ -290,3 +302,15 @@ Style/OptionalBooleanParameter:
 
 Style/StringConcatenation:
   Enabled: false
+
+Style/CombinableLoops:
+  Enabled: true
```

Comment:
Feels like this might have some unintended consequences, but happy to turn it on and find out...

Reply:
I tested this branch against PS, this cop only breaks once so we should be fine